source/trx2junit/Internal/JUnitBuilder.cs

```diff
@@ -31,6 +31,12 @@ public void Build()
         //---------------------------------------------------------------------
         private void AddTestSuite(string testSuiteName, IEnumerable<KeyValuePair<Guid, TestDefinition>> tests)
         {
+            _errors = 0;
```

Comment:
Can you move this to a separate method `ResetCounters` (or similar)?

Reply:
Sure, makes it cleaner